lib/src/license_detection/crc32.dart

```diff
@@ -0,0 +1,278 @@
+/// Computes CRC-32 checksum for [bytes] supplied. Returns unsigned 32 bit integer.
+///
+/// Specification of [CRC-32][1] is same as used in IEEE 802.3 Ethernet (Normal).
+/// Width  : 32
+/// Poly   : 0x04c11db7
+/// Init   : 0xffffffff
+/// RefIn  : true
+/// RefOut : true
+/// [1]: https://en.wikipedia.org/wiki/Cyclic_redundancy_check
```

Comment:
If you want an ascii table wrap it in a code fences... 
```suggestion
/// Specification of [CRC-32][1] is same as used in IEEE 802.3 Ethernet (Normal).
/// ```
/// Width  : 32
/// Poly   : 0x04c11db7
/// Init   : 0xffffffff
/// RefIn  : true
/// RefOut : true
/// ```
///
/// [1]: https://en.wikipedia.org/wiki/Cyclic_redundancy_check
```